client/src/components/VideoPlayer.jsx

```diff
@@ -6,21 +6,45 @@ const VideoTitle = styled.p`
   text-transform: uppercase;
   font-family: Helvetica;
 `
+const PlayerImage = styled.img`
+  width: 80%;
+`
 
 class VideoPlayer extends React.Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      isClicked: false
+    }
+  }
+
+  handleImgClick() {
+    this.setState({
+      isClicked: true
+    })
   }
 
   render() {
     //"https://www.youtube.com/watch?v=UMaAF2WTK1A" --> "https://www.youtube.com/embed/UMaAF2WTK1A"
     let youtubeURL = "https://www.youtube.com/embed/";
     let vidID = this.props.video.url.split('=')[1];
 
-    return (
-      <div className="videoPlayer">
+    let videoDisplay;
+
+    if (this.state.isClicked) {
+      videoDisplay = (
       <iframe width="1200" height="700" src={`${youtubeURL}${vidID}?autoplay=0&controls=0`} frameBorder="0">
       </iframe>
+      )
+    } else {
+      videoDisplay = (
+      <PlayerImage src={`http://img.youtube.com/vi/${vidID}/hqdefault.jpg`} onClick={() => {this.handleImgClick()}}/>)
```

Comment:
I haven't written references to methods as an inline function like this: onClick={() => {this.handleImgClick()}}
I see how this would work, though. Have you tried it like this: onClick={this.handleImgClick} ?

Reply:
If that still works, I think it would be more readable.

Reply:
Thanks Daniel! I think that syntax would almost work, it just would need the 'this' binding added-- `this.handleImgClick.bind(this)` I think you're right that it's more readable, even with the binding syntax added.